services/datamanager/builtin/capture_test.go

```diff
@@ -31,6 +31,8 @@ var (
 	remoteCollectorConfigPath                   = "services/datamanager/data/fake_robot_with_remote_and_data_manager.json"
 	emptyFileBytesSize                          = 100 // size of leading metadata message
 	captureInterval                             = time.Millisecond * 10
+	initialJmagValue                            = float64(1)
```

Comment:
I think these are only used in the one test - can you move them to within that test to minimize scope?